### ISSUE TITLE
You must add a reference to assembly 'netstandard' issue

### ISSUE THREAD

**Issue by danhastings** [kblok/puppeteer-sharp#513]
I have a class library project with a targeting .NET Framework 4.6.2. Without writing any code and simply referencing the nuget package the project builds just fine. Once I write a wrapper class for puppeteer i get loads of netstandard exceptions. 

CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'. in PuppeteerSharpService.cs(19, 40)
CS0012: The type 'Task<>' is defined in an assembly that is not referenced. You must add a reference to assembly 'netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'. in PuppeteerSharpService.cs(19, 19)
CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'. in PuppeteerSharpService.cs(20, 29)
CS0012: The type 'Task<>' is defined in an assembly that is not referenced. You must add a reference to assembly 'netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'. in PuppeteerSharpService.cs(20, 19)
.
.
.

The constructor doesn't have anything fancy. I have no references to this class anywhere in the project. Just adding this code to the class causes it to throw a while load of exceptions like the ones above. 
```
public PuppeteerSharpService()
{
	var bfetcher = new BrowserFetcher().DownloadAsync(BrowserFetcher.DefaultRevision).Result;
	this.Browser = Puppeteer.LaunchAsync(new LaunchOptions
	{
		Headless = true
	}).Result;
	this.Page = this.Browser.NewPageAsync().Result;
}
```
The class uses the following assemblies. 

using PuppeteerSharp;
using System.Collections.Generic;
using System.Net;

Any idea what Is going wrong here?

**Comment by kblok** [comment]
Could you take a look at this link and tell me if it helped you https://github.com/dotnet/standard/issues/391?

**Comment by danhastings** [comment]
I have tried some of these steps already. NETStandard.Library.NETFramework has since been deprecated. It seems upgrading to the latest version of the .NET framework will resolve this but I don't have the ability to do this at the minute. 

It seems weird that the NETStandard library didnt get installed automatically when I installed the puppeteer library. I tried installing it anyway and nothing happened.
